### DISCUSSION TITLE
How to get variable at another column in the table?

### DISCUSSION THREAD

**Discussion by amantiwari1** [vueComponent/ant-design-vue#4519] [Q&A] [answered]
for example

 ```vue
 <p slot="instance_name" slot-scope="instance_name">
              {{ instance_name }} 
              I want to get status for this 
              {{status}} it seems doesn't works 
 </p>
 <p slot="status" slot-scope="status">{{ status }}</p>
 ```
 
 why status? because I want to check status like if status is failed then `instance_name ` text  will be red 
 and vice versa `instance_name ` text will be green

**Comment by zkwolf** [comment, marked as answer]
you can use record, like the aciton column in https://antdv.com/components/table/#basic-Usage
